kasket.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.license     = "Apache License Version 2.0"
   s.files       = Dir.glob("lib/**/*") + %w[README.md]
 
-  s.required_ruby_version = '>= 2.6.0'
+  s.required_ruby_version = '>= 2.7'
 
-  s.add_runtime_dependency("activerecord", ">= 5.1", "< 7")
+  s.add_runtime_dependency("activerecord", ">= 5.1", "< 7.1")
```

Comment:
Should we consider removing the upper limit on ActiveRecord? Or do we have too many monkey patches to assume a new AR release would “just work” with Kasket?

Reply:
I... don't know. This is a good question and one we've discussed in slack. Since it is a public gem I'd be hesitant to remove the top completely, but also, I pity the fool who uses this outside our org. 🤪

Buuut... at 224 downloads since April(!), I suspect there aren't any. So that's a good sign that we can be a little less conservative.

Reply:
My vote would be to keep the upper limit on ActiveRecord for the time being. Instead, I'd like to suggest that we mark this library as deprecated and no longer maintained by us. I don't see us maintaining/supporting it post Rails 7.